thoth/storages/graph/postgres.py

```diff
@@ -4413,8 +4413,8 @@ def _check_package_solved(self, session: Session, package_name: str, package_ver
     def sync_security_indicator_aggregated_result(self, document: dict) -> None:
         """Sync the given security-indicator aggregated result to the graph database."""
         metadata = document["metadata"]
-        result = document["result"]
         document_id = metadata["document_id"]
+        result = document.get("result", dict())
```

Comment:
isn't it inside `metadata`? https://github.com/thoth-station/si-aggregator/pull/35/files#

Reply:
@KPostOffice  Please also adjust queries related to retrieve of packages to be si analyzed, so those with errors are not retrieved.